terraform/datadog_email_activity.tf

```diff
@@ -0,0 +1,458 @@
+      source_type          = "attribute"
+      target               = "user-role"
+      target_type          = "tag"
+      target_format        = "string"
+      preserve_source      = true
+      override_on_conflict = false
+      name                 = "Map mail.tags.user_role to 'user-role' tag"
+      is_enabled           = true
+    }
+  }
+  processor {
+    attribute_remapper {
+      sources              = ["mail.tags.team_id"]
+      source_type          = "attribute"
+      target               = "team-id"
+      target_type          = "tag"
+      target_format        = "string"
+      preserve_source      = true
+      override_on_conflict = false
+      name                 = "Map mail.tags.team_id to 'team-id' tag"
+      is_enabled           = true
+    }
+  }
+  processor {
+    attribute_remapper {
+      sources              = ["mail.tags.organization_id"]
+      source_type          = "attribute"
+      target               = "organization-id"
+      target_type          = "tag"
+      target_format        = "string"
+      preserve_source      = true
+      override_on_conflict = false
+      name                 = "Map mail.tags.organization_id to 'organization-id' tag"
+      is_enabled           = true
+    }
+  }
+  processor {
+    attribute_remapper {
+      sources              = ["mail.tags.ses:configuration-set"]
+      source_type          = "attribute"
+      target               = "configuration-set"
+      target_type          = "tag"
+      target_format        = "string"
+      preserve_source      = true
+      override_on_conflict = false
+      name                 = "Map mail.tags.ses:configuration-set to 'configuration-set' tag"
+      is_enabled           = true
+    }
+  }
+  processor {
+    attribute_remapper {
+      sources              = ["mail.tags.version"]
+      source_type          = "attribute"
+      target               = "version"
+      target_type          = "tag"
+      target_format        = "string"
+      preserve_source      = true
+      override_on_conflict = false
+      name                 = "Map mail.tags.version to 'version' tag"
+      is_enabled           = true
+    }
+  }
+}
+
+resource "datadog_logs_metric" "gost.ses.email_sending_event.subscription" {
+  name = "gost.ses.email_sending_event.subscription"
+  compute {
+    aggregation_type = "count"
+  }
+  filter {
+    query = "@Sns.Subject:'Amazon SES Email Event Notification' @eventType:Subscription service:gost"
+  }
+  group_by {
+    path     = "@mail.tags.notification_type"
+    tag_name = "notification-type"
+  }
+  group_by {
+    path     = "@mail.tags.organization_id"
+    tag_name = "organization-id"
+  }
+  group_by {
+    path     = "@mail.tags.ses:configuration-set"
+    tag_name = "configuration-set"
+  }
+  group_by {
+    path     = "@mail.tags.team_id"
+    tag_name = "team-id"
+  }
+  group_by {
+    path     = "@mail.tags.organization_id"
+    tag_name = "organization-id"
+  }
```

Comment:
Looks like this `group_by` is duplicated – there already appears to be a `group_by` block targeting the `organization-id` tag a few lines above.

I also see this repeated on the other `datadog_logs_metric` resources.

Reply:
Thanks for the catch! Totally missed that.